src/rocm_docs/core.py

```diff
@@ -46,7 +46,7 @@ def update_body(body: Dict[str, Any]) -> None:
 
 @_config_updater
 def _ConfigExtend(key: str, app: Sphinx, default: T) -> None:
-    getattr(app.config, key).extend(default)
+    list(getattr(app.config, key)).extend(default)
```

Comment:
This won't have the desired effect, you're just creating a new list modifying it and throwing it away.

`myst_enable_extensions` is now a set, see [myst_parser configuration options](https://myst-parser.readthedocs.io/en/latest/configuration.html#global-configuration). You should change the value on line 79 to a set and introduce a helper for merging sets something like
```python
@_config_updater
def _ConfigUnion(key: str, app: Sphinx, default: Set[T]) -> None:
    gettattr(app.config, key).update(default)
```

Reply:
Thanks for pointing that out.